docs/General/on-the-server.rst

```diff
@@ -0,0 +1,109 @@
+..
+    SPDX-License-Identifier: GPL-3.0-or-later
+    SPDX-FileCopyrightText: 2022 louis94 <m_louis30@yahoo.com>
+
+Freeciv21 on the Server
+***********************
+
+In order to play Freeciv21 on the network, one machine needs to act as the *server*: the state of
+the game lives on the server and players can connect to play. There are many ways to operate such a
+server; this page gives an introduction to the topic.
+
+In order to run a server, you need a computer that other players can reach over the network. If
+you're planning to play at home, this can usually be any machine connected to your WiFi or local
+cabled network. Otherwise, the easiest is to rent a small server from a hosting provider. We
+strongly recommend that you choose a Linux-based server, as that is what we have experience with.
+You will need the ability to run your own programs on the server, so SSH access is a must. Apart
```

Comment:
SSH is a program, so wrap in two backticks or use :file: directive.

Reply:
SSH stands for *Secure Shell* and is not usually set in monospace in its uppercase form. It's also not a `:file:` because the file is called `ssh` (lowercase).

Reply:
Yea that makes sense.